Cargo.toml

```diff
@@ -12,37 +12,37 @@ readme = "README.md"
 resolver = "2"
 
 [features]
-default = ["u64_backend", "serialize"]
+default = ["ristretto255_u64", "serde"]
+ristretto255_u64 = ["curve25519-dalek/u64_backend"]
+ristretto255_u32 = ["curve25519-dalek/u32_backend"]
+ristretto255_fiat_u64 = ["curve25519-dalek/fiat_u64_backend"]
+ristretto255_fiat_u32 = ["curve25519-dalek/fiat_u32_backend"]
+ristretto255_simd = ["curve25519-dalek/simd_backend"]
 p256 = ["num-bigint", "num-integer", "num-traits", "once_cell", "p256_"]
-std = ["curve25519-dalek/std", "getrandom", "rand/std", "rand/std_rng", "num-bigint/std", "num-integer/std", "num-traits/std"]
-u64_backend = ["curve25519-dalek/u64_backend"]
-u32_backend = ["curve25519-dalek/u32_backend"]
-simd_backend = ["curve25519-dalek/simd_backend"]
-serialize = ["serde", "base64", "generic-array/serde", "curve25519-dalek/serde"]
+std = []
+serde = ["serde_", "base64"]
```

Comment:
I renamed `serialize` to `serde` because I realized that `serialization` is available without this feature. It's very common to name this crate feature `serde` anyway.